### PR TITLE
Added gmf_ebrisk case_4

### PR DESCRIPTION
**Pull request by micheles**
This is a simple test with a single asset and only three ruptures but two source models. It is meant to help debugging the issue https://github.com/gem/oq-engine/issues/3318.

### CODE REVIEW

File: openquake/calculators/tests/gmf_ebrisk_test.py [hunk 1]
```diff
@@ -79,6 +80,26 @@ def test_ebr_2(self):
...
+                      calculation_mode='event_based_risk',
+                      hazard_calculation_id=str(calc0.calc_id))
+        calc1 = self.calc.datastore  # event_based_risk
+        self.run_calc(case_4.__file__, 'job_risk.ini',
+                      hazard_calculation_id=str(calc0.calc_id))
+        calc2 = self.calc.datastore  # event_based_risk
+        [fname] = export(('agg_loss_table', 'csv'), calc1)
+        self.assertEqualFiles('expected/' + strip_calc_id(fname), fname,
+                              delta=1E-5)
+        #[fname] = export(('agg_loss_table', 'csv'), calc2)
```
Comment (ptormene) [inline openquake/calculators/tests/gmf_ebrisk_test.py]: Can this be removed?

Comment (micheles) [reply openquake/calculators/tests/gmf_ebrisk_test.py]: This is the part that it is breaking the test and that I will fix in the future PR. The whole purpose of this PR is to expose this issue ;-)

File: openquake/qa_tests_data/gmf_ebrisk/case_4/job_haz.ini [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+[logic_trees]
+source_model_logic_tree_file = source_model_logic_tree.xml
+gsim_logic_tree_file = gsim_logic_tree.xml
+
+[correlation]
+ground_motion_correlation_model = JB2009
+ground_motion_correlation_params = {"vs30_clustering": True}
+
+[hazard_calculation]
+#intensity_measure_types = PGA, SA(0.1), SA(0.3), SA(0.4)
```
Comment (ptormene) [inline openquake/qa_tests_data/gmf_ebrisk/case_4/job_haz.ini]: Should this be removed?

Comment (micheles) [reply openquake/qa_tests_data/gmf_ebrisk/case_4/job_haz.ini]: Oh, yes, this is a leftover, I missed that!

**Pull request merged**